src/sentry/stacktraces/processing.py

```diff
@@ -543,6 +574,15 @@ def process_stacktraces(data, make_processors=None, set_raw_stacktrace=True):
                     stacktrace_info.stacktrace["frames"] = new_frames
                     changed = True
                     span.set_data("data_changed", True)
+
+                mechanism = stacktrace_info.container.get("mechanism") if is_exception else None
+                error_type = stacktrace_info.container.get("type") if is_exception else None
+                with sentry_sdk.configure_scope() as scope:
+                    if JAVASCRIPT_PLATFORMS in stacktrace_info.platforms:
```

Comment:
Since this is javascript specific, I'm wondering if we should put this logic here instead: https://github.com/getsentry/sentry/blob/4ccd5c49d28c5124a1cc3289742cdc9219ff7448/src/sentry/lang/javascript/processor.py#L811

Reply:
Done :)